Examples/MagicWeather/MagicWeather/Constants.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Constants.swift
+//  Magic Weather
+//
+//  Created by Cody Kerns on 12/14/20.
+//
+
+import Foundation
+
+/*
+ Configuration file for your app's RevenueCat settings.
+ */
+
+struct Constants {
+    
+    /*
+     The API key for your app from the RevenueCat dashboard: https://app.revenuecat.com
+     */
+    #error("Modify this property to reflect your app's API key, then comment this line out.")
+    static let apiKey = "api_key"
+    
+    /*
+     The entitlement ID that is activated upon successful in-app purchase.
+     */
+    #warning("Modify this property to reflect your app's entitlement.")
```

Comment:
Should we make this an #error too? It would break the app if not set up properly.